# Change Log

## [Unreleased](https://github.com/sanger-pathogens/update_pipeline/tree/HEAD)

[Full Changelog](https://github.com/sanger-pathogens/update_pipeline/compare/v0.0.1...HEAD)

**Merged pull requests:**

- Include tests in README [\#107](https://github.com/sanger-pathogens/update_pipeline/pull/107) ([ssjunnebo](https://github.com/ssjunnebo))
- Update README [\#106](https://github.com/sanger-pathogens/update_pipeline/pull/106) ([ssjunnebo](https://github.com/ssjunnebo))
- 574328 fix travis tests [\#105](https://github.com/sanger-pathogens/update_pipeline/pull/105) ([ssjunnebo](https://github.com/ssjunnebo))
- remove version nr, autoversion instead [\#104](https://github.com/sanger-pathogens/update_pipeline/pull/104) ([ssjunnebo](https://github.com/ssjunnebo))

## [v0.0.1](https://github.com/sanger-pathogens/update_pipeline/tree/v0.0.1) (2018-01-25)
**Merged pull requests:**

- Add version number [\#103](https://github.com/sanger-pathogens/update_pipeline/pull/103) ([ssjunnebo](https://github.com/ssjunnebo))
- Fix Travis Errors [\#102](https://github.com/sanger-pathogens/update_pipeline/pull/102) ([ssjunnebo](https://github.com/ssjunnebo))
- change order of imeta query to speed things up [\#101](https://github.com/sanger-pathogens/update_pipeline/pull/101) ([andrewjpage](https://github.com/andrewjpage))
- add run\_date from mlwarehouse [\#100](https://github.com/sanger-pathogens/update_pipeline/pull/100) ([andrewjpage](https://github.com/andrewjpage))
- Get insert size from new warehouse [\#99](https://github.com/sanger-pathogens/update_pipeline/pull/99) ([andrewjpage](https://github.com/andrewjpage))
- add hash to check common name [\#98](https://github.com/sanger-pathogens/update_pipeline/pull/98) ([ssjunnebo](https://github.com/ssjunnebo))
- update email address [\#97](https://github.com/sanger-pathogens/update_pipeline/pull/97) ([ssjunnebo](https://github.com/ssjunnebo))
- dont import if there is no run id [\#96](https://github.com/sanger-pathogens/update_pipeline/pull/96) ([andrewjpage](https://github.com/andrewjpage))
- filter out old pb lanes [\#95](https://github.com/sanger-pathogens/update_pipeline/pull/95) ([andrewjpage](https://github.com/andrewjpage))
- inconsistent naming of library [\#94](https://github.com/sanger-pathogens/update_pipeline/pull/94) ([andrewjpage](https://github.com/andrewjpage))
- populate library name and sample public name if missing [\#93](https://github.com/sanger-pathogens/update_pipeline/pull/93) ([andrewjpage](https://github.com/andrewjpage))
- if library or sample missing fill in with placeholder data [\#92](https://github.com/sanger-pathogens/update_pipeline/pull/92) ([andrewjpage](https://github.com/andrewjpage))
- Pacbio duplicate libraries [\#91](https://github.com/sanger-pathogens/update_pipeline/pull/91) ([andrewjpage](https://github.com/andrewjpage))
- infgen [\#90](https://github.com/sanger-pathogens/update_pipeline/pull/90) ([andrewjpage](https://github.com/andrewjpage))
- update from 108 to 118 [\#89](https://github.com/sanger-pathogens/update_pipeline/pull/89) ([andrewjpage](https://github.com/andrewjpage))
- SRA fastq-dump header replacing [\#88](https://github.com/sanger-pathogens/update_pipeline/pull/88) ([vaofford](https://github.com/vaofford))
- add symlink rsync [\#87](https://github.com/sanger-pathogens/update_pipeline/pull/87) ([vaofford](https://github.com/vaofford))
- lookup xml directly first before using bioperl [\#86](https://github.com/sanger-pathogens/update_pipeline/pull/86) ([andrewjpage](https://github.com/andrewjpage))
- ncbi changed to https [\#85](https://github.com/sanger-pathogens/update_pipeline/pull/85) ([andrewjpage](https://github.com/andrewjpage))
- skip missing common name [\#84](https://github.com/sanger-pathogens/update_pipeline/pull/84) ([andrewjpage](https://github.com/andrewjpage))
- filter out lanes without study name [\#83](https://github.com/sanger-pathogens/update_pipeline/pull/83) ([andrewjpage](https://github.com/andrewjpage))
- pacbio should draw in results from irods [\#82](https://github.com/sanger-pathogens/update_pipeline/pull/82) ([andrewjpage](https://github.com/andrewjpage))
- nds\_change\_to\_dist\_zilla\_starter\_bundle [\#81](https://github.com/sanger-pathogens/update_pipeline/pull/81) ([nds](https://github.com/nds))
- bin directory should be set by softwarerc [\#80](https://github.com/sanger-pathogens/update_pipeline/pull/80) ([andrewjpage](https://github.com/andrewjpage))
- remove blank header cells [\#79](https://github.com/sanger-pathogens/update_pipeline/pull/79) ([andrewjpage](https://github.com/andrewjpage))
- Long running validate reports failing [\#78](https://github.com/sanger-pathogens/update_pipeline/pull/78) ([andrewjpage](https://github.com/andrewjpage))
- Fix for validate GCLP warehouse [\#77](https://github.com/sanger-pathogens/update_pipeline/pull/77) ([andrewjpage](https://github.com/andrewjpage))
- Data access group [\#76](https://github.com/sanger-pathogens/update_pipeline/pull/76) ([andrewjpage](https://github.com/andrewjpage))
- expose threads option for spreadsheet [\#75](https://github.com/sanger-pathogens/update_pipeline/pull/75) ([andrewjpage](https://github.com/andrewjpage))
- Run date [\#74](https://github.com/sanger-pathogens/update_pipeline/pull/74) ([andrewjpage](https://github.com/andrewjpage))
- update warehouse 2 cname [\#73](https://github.com/sanger-pathogens/update_pipeline/pull/73) ([andrewjpage](https://github.com/andrewjpage))
- only print sync message when verbose on [\#72](https://github.com/sanger-pathogens/update_pipeline/pull/72) ([andrewjpage](https://github.com/andrewjpage))
- Increase stability [\#71](https://github.com/sanger-pathogens/update_pipeline/pull/71) ([andrewjpage](https://github.com/andrewjpage))
- Reduce unnessisary lookups for warehouse fragment size [\#70](https://github.com/sanger-pathogens/update_pipeline/pull/70) ([andrewjpage](https://github.com/andrewjpage))
- cram file type in validation [\#69](https://github.com/sanger-pathogens/update_pipeline/pull/69) ([andrewjpage](https://github.com/andrewjpage))
- filter rows [\#68](https://github.com/sanger-pathogens/update_pipeline/pull/68) ([andrewjpage](https://github.com/andrewjpage))
- make sure there are no duplicate filenames [\#67](https://github.com/sanger-pathogens/update_pipeline/pull/67) ([andrewjpage](https://github.com/andrewjpage))
- Speedup md5 [\#66](https://github.com/sanger-pathogens/update_pipeline/pull/66) ([andrewjpage](https://github.com/andrewjpage))
- least specific to most specific speeds up irods query [\#65](https://github.com/sanger-pathogens/update_pipeline/pull/65) ([andrewjpage](https://github.com/andrewjpage))
- change target param to end of search [\#64](https://github.com/sanger-pathogens/update_pipeline/pull/64) ([andrewjpage](https://github.com/andrewjpage))
- default to bam [\#63](https://github.com/sanger-pathogens/update_pipeline/pull/63) ([andrewjpage](https://github.com/andrewjpage))
- add support to import cram files [\#62](https://github.com/sanger-pathogens/update_pipeline/pull/62) ([andrewjpage](https://github.com/andrewjpage))
- pull in reference [\#61](https://github.com/sanger-pathogens/update_pipeline/pull/61) ([andrewjpage](https://github.com/andrewjpage))
- import all QC'd bams [\#60](https://github.com/sanger-pathogens/update_pipeline/pull/60) ([sb10](https://github.com/sb10))
- fix syntax for study lookup [\#59](https://github.com/sanger-pathogens/update_pipeline/pull/59) ([andrewjpage](https://github.com/andrewjpage))
- update lane [\#58](https://github.com/sanger-pathogens/update_pipeline/pull/58) ([andrewjpage](https://github.com/andrewjpage))
- missing data in irods so use warehouse primarily [\#57](https://github.com/sanger-pathogens/update_pipeline/pull/57) ([andrewjpage](https://github.com/andrewjpage))
- Pacbio err numbers [\#56](https://github.com/sanger-pathogens/update_pipeline/pull/56) ([andrewjpage](https://github.com/andrewjpage))
- Reduce memory usage and move to dist zilla [\#55](https://github.com/sanger-pathogens/update_pipeline/pull/55) ([andrewjpage](https://github.com/andrewjpage))
- Update ERR numbers from irods [\#54](https://github.com/sanger-pathogens/update_pipeline/pull/54) ([andrewjpage](https://github.com/andrewjpage))
- Added AUTHORS file [\#53](https://github.com/sanger-pathogens/update_pipeline/pull/53) ([aslett1](https://github.com/aslett1))
- resolve and merge vertres changes  [\#52](https://github.com/sanger-pathogens/update_pipeline/pull/52) ([andrewjpage](https://github.com/andrewjpage))
- lower default num reads [\#51](https://github.com/sanger-pathogens/update_pipeline/pull/51) ([andrewjpage](https://github.com/andrewjpage))
- Pac bio metadata tracking [\#49](https://github.com/sanger-pathogens/update_pipeline/pull/49) ([andrewjpage](https://github.com/andrewjpage))
- all tests now pass [\#48](https://github.com/sanger-pathogens/update_pipeline/pull/48) ([andrewjpage](https://github.com/andrewjpage))
- added --species option to avoid NCBI database look up [\#47](https://github.com/sanger-pathogens/update_pipeline/pull/47) ([sb10](https://github.com/sb10))
- -md5 and -use\_supplier\_name fixes [\#46](https://github.com/sanger-pathogens/update_pipeline/pull/46) ([sb10](https://github.com/sb10))
- Changes made to vertebrate resequencing update pipeline \(updated\) [\#45](https://github.com/sanger-pathogens/update_pipeline/pull/45) ([sb10](https://github.com/sb10))
- Changes made to vertebrate resequencing update pipeline [\#43](https://github.com/sanger-pathogens/update_pipeline/pull/43) ([jm23](https://github.com/jm23))
- set mysql\_auto\_connect [\#42](https://github.com/sanger-pathogens/update_pipeline/pull/42) ([CraigPorter](https://github.com/CraigPorter))
- added confirm completed message [\#41](https://github.com/sanger-pathogens/update_pipeline/pull/41) ([CraigPorter](https://github.com/CraigPorter))
- pull fragment size for wells from batch requests [\#39](https://github.com/sanger-pathogens/update_pipeline/pull/39) ([andrewjpage](https://github.com/andrewjpage))
- GPL [\#38](https://github.com/sanger-pathogens/update_pipeline/pull/38) ([andrewjpage](https://github.com/andrewjpage))
- update tests for fastq direction [\#36](https://github.com/sanger-pathogens/update_pipeline/pull/36) ([andrewjpage](https://github.com/andrewjpage))
- Set readlength, reads and bases when importing using spreadsheet. [\#35](https://github.com/sanger-pathogens/update_pipeline/pull/35) ([CraigPorter](https://github.com/CraigPorter))
- If irods manual\_qc not set, check warehouse [\#34](https://github.com/sanger-pathogens/update_pipeline/pull/34) ([andrewjpage](https://github.com/andrewjpage))
- replaced listallmissing option with no\_pending\_lanes [\#33](https://github.com/sanger-pathogens/update_pipeline/pull/33) ([CraigPorter](https://github.com/CraigPorter))
- added option to list missing lanes if qc pending [\#32](https://github.com/sanger-pathogens/update_pipeline/pull/32) ([CraigPorter](https://github.com/CraigPorter))
- list missing lanes after qc complete [\#31](https://github.com/sanger-pathogens/update_pipeline/pull/31) ([CraigPorter](https://github.com/CraigPorter))
- Lock file to prevent duplicate processes on the same host [\#30](https://github.com/sanger-pathogens/update_pipeline/pull/30) ([andrewjpage](https://github.com/andrewjpage))
- update warehouse host [\#29](https://github.com/sanger-pathogens/update_pipeline/pull/29) ([andrewjpage](https://github.com/andrewjpage))
- Spreadsheet changes [\#28](https://github.com/sanger-pathogens/update_pipeline/pull/28) ([aslett1](https://github.com/aslett1))
- New SimpleLookup for species name and changed tags for optional parameters [\#27](https://github.com/sanger-pathogens/update_pipeline/pull/27) ([jm23](https://github.com/jm23))
- Set file types for forward and reverse fastqs [\#26](https://github.com/sanger-pathogens/update_pipeline/pull/26) ([CraigPorter](https://github.com/CraigPorter))
- new warehouse [\#25](https://github.com/sanger-pathogens/update_pipeline/pull/25) ([andrewjpage](https://github.com/andrewjpage))
- ichksum format changed [\#24](https://github.com/sanger-pathogens/update_pipeline/pull/24) ([andrewjpage](https://github.com/andrewjpage))
- Md5 fastqcheck spreadsheet [\#23](https://github.com/sanger-pathogens/update_pipeline/pull/23) ([andrewjpage](https://github.com/andrewjpage))
- create fastqcheck file and md5 [\#22](https://github.com/sanger-pathogens/update_pipeline/pull/22) ([andrewjpage](https://github.com/andrewjpage))
- Changes for vertres tracking pipeline [\#21](https://github.com/sanger-pathogens/update_pipeline/pull/21) ([jm23](https://github.com/jm23))
- Add no moose and make meta immutable [\#20](https://github.com/sanger-pathogens/update_pipeline/pull/20) ([mp15](https://github.com/mp15))
- add a random number to hierarchy name in individual if clash [\#19](https://github.com/sanger-pathogens/update_pipeline/pull/19) ([andrewjpage](https://github.com/andrewjpage))
- dont add common name by default [\#18](https://github.com/sanger-pathogens/update_pipeline/pull/18) ([andrewjpage](https://github.com/andrewjpage))
- Updated spreadsheet template for remote users [\#17](https://github.com/sanger-pathogens/update_pipeline/pull/17) ([aslett1](https://github.com/aslett1))
- spreadsheet tempalte [\#16](https://github.com/sanger-pathogens/update_pipeline/pull/16) ([andrewjpage](https://github.com/andrewjpage))
- Spreadsheet import [\#15](https://github.com/sanger-pathogens/update_pipeline/pull/15) ([andrewjpage](https://github.com/andrewjpage))
- command line option to activate read count check [\#14](https://github.com/sanger-pathogens/update_pipeline/pull/14) ([fy2](https://github.com/fy2))
- Validation has been extended [\#13](https://github.com/sanger-pathogens/update_pipeline/pull/13) ([fy2](https://github.com/fy2))
- merge changes from jm23 [\#12](https://github.com/sanger-pathogens/update_pipeline/pull/12) ([andrewjpage](https://github.com/andrewjpage))
- update warehouse sql [\#10](https://github.com/sanger-pathogens/update_pipeline/pull/10) ([andrewjpage](https://github.com/andrewjpage))
- warehouse two sql schema [\#9](https://github.com/sanger-pathogens/update_pipeline/pull/9) ([andrewjpage](https://github.com/andrewjpage))
- lookup study ssid from seqscape tables first to avoid data problems with... [\#8](https://github.com/sanger-pathogens/update_pipeline/pull/8) ([andrewjpage](https://github.com/andrewjpage))
- validate [\#7](https://github.com/sanger-pathogens/update_pipeline/pull/7) ([andrewjpage](https://github.com/andrewjpage))
- Mean insert size [\#6](https://github.com/sanger-pathogens/update_pipeline/pull/6) ([andrewjpage](https://github.com/andrewjpage))
- delete files when lane reset [\#5](https://github.com/sanger-pathogens/update_pipeline/pull/5) ([andrewjpage](https://github.com/andrewjpage))
- Build vr track [\#4](https://github.com/sanger-pathogens/update_pipeline/pull/4) ([andrewjpage](https://github.com/andrewjpage))
- reduce sensitivity [\#3](https://github.com/sanger-pathogens/update_pipeline/pull/3) ([andrewjpage](https://github.com/andrewjpage))
- tweaks for error email [\#2](https://github.com/sanger-pathogens/update_pipeline/pull/2) ([andrewjpage](https://github.com/andrewjpage))
- Report which flags up problematic lanes in the pipelines [\#1](https://github.com/sanger-pathogens/update_pipeline/pull/1) ([andrewjpage](https://github.com/andrewjpage))

